docs/src/lens.md

```diff
@@ -137,7 +137,7 @@ impl Lens<Container2, Name> for NameLens {
 }
 ```
 
-Now in addition to almost free `Clone`s, we also have cheap incremental updates to the data itself. In the case of names, this isn't that important, but if the vector had `1_000_000_000` elements, we could still make changes in only *O(log(n))* time (in this case the difference between `1_000_000_000` and `30` - pretty big!).
+Now in addition to almost free `Clone`s, we also have cheap incremental updates to the data itself. In the case of names, this isn't that important, but if the vector had `1_000_000_000` elements, we could still make changes in only *O(log(n))* time (in this case the difference between `1_000_000_000` and `9` - pretty big!).
```

Comment:
mostly, we use log base 2? in any case, this should be a separate PR

Reply:
some discussion: https://github.com/linebender/druid/pull/1444#discussion_r535253139

Reply:
Close the PR since everything is redundant with #1444.
Sorry for the inconvenience.

Reply:
typo `valid` needs to be fixed

Reply:
I removed the last commit to keep only the relevant typo.